kratos/utilities/variable_utils.h

```diff
@@ -384,11 +384,33 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
         KRATOS_CATCH("")
     }
 
+
+    /**
+     * @brief Clears the container data value container
+     * @param rContainer Reference to the objective container
+     */
+    template< class TContainerType>
+    void ClearNonHistoricalValues(TContainerType& rContainer)
```

Comment:
could you add the same for historical vars (just for nodes ofc) for completeness?

Reply:
I don't know, I am not sure if it is possible to clear the database

Reply:
I will check, but I prefer to let that for a future PR

Reply:
ok

Another thing that is a bit ambiguous imo is that I though that this function sets all the non-hist vars to zero, but it does sth else.

How abt `ClearNonHistoricalData`? then it does exactly what the name indicates

Reply:
That's true